modules/iopcore/cdvdman/device-mmce.c

```
@@ -0,0 +1,189 @@
+    // Judge the LAST attempt, not the retry counter: a read that succeeds on the 3rd try also leaves
+    // retries == 3, and the old counter check misreported that success as SCECdErREAD.
+    if (res != sectors) {
+        DPRINTF("%s: Failed to read after %d retries, sector: %u, count: %u, buffer: 0x%p\n", __func__, retries, (unsigned int)lsn, sectors, buffer);
+        rv = SCECdErREAD;
+    }
+
+    return rv;
+}
+
+//TODO: For VMCs
+int mmce_read_offset(int fd, unsigned int offset, unsigned int size, unsigned char *buffer)
```

Cppcheck (2.21.0)

[style] The function 'mmce_read_offset' is never used.

(unusedFunction)